tasks/install-enketo.yml

```diff
@@ -143,6 +143,20 @@
   notify:
     - "reload pm2-{{ enketo_user }}"
 
+- block:
+    - name: Replace pm2 resurrect with pm2 start
+      lineinfile:
+        regexp: "^ExecStart"
+        line: "ExecStart={{ enketo_pm2_path }} start app.js -n {{ enketo_user }} -i 0"
+        dest: "/etc/systemd/system/pm2-enketo.service"
```

Comment:
How about /etc/systemd/system/pm2-{{ enketo_user }}.service since, I think, the name of the service is determined by the value of the enketo_user variable.